examples/example_config.py

```diff
@@ -33,7 +33,8 @@
 #
 #   worker_class - The type of workers to use. The default
 #       async class should handle most 'normal' types of work
-#       loads. You'll want to read http://gunicorn/deployment.hml
+#       loads. You'll want to read
+#       http://docs.gunicorn.org/en/latest/design.html
```

Comment:
http://docs.gunicorn.org/en/latest/design.html#choosing-a-worker-type would be a better URL.